src/index.js

```diff
@@ -133,7 +133,8 @@ export {
   TabPane,
 } from './Tabs';
 export { default as TextArea } from './TextArea';
-export { default as Toast, TOAST_CLOSE_LABEL_TEXT, TOAST_DELAY } from './Toast';
+export { default as ToastContainer } from './Toast/ToastContainer';
+export { toast } from './Toast/toast';
```

Comment:
[important] I suggest changing the file structure in the Toast folder.

Let's consider the option of moving the `EventEmitter.js` and `toast.js` files to the `utils` folder (+ we will implement re-export for these files). Let's rename the `index.jsx` file to `BaseToast`, and `ToastContainer.jsx` in `index.jsx`.

**For example:**
- Toast
  - tests
    - ...
  - utils
    - EventEmitter.js
    - index.js
    - toast.js
  - _variables.scss
  - BaseToast.jsx
  - index.scss
  - README.MD
  - index.jsx

**The export from Paragon will look like this:**
`export { default as ToastContainer, toast } from './Toast';`